op-bindings/artifacts.json

```diff
@@ -46,5 +46,104 @@
     "ISemver",
     "StorageSetter",
     "SuperchainConfig"
+  ],
+  "remote": [
+    {
+      "name": "MultiCall3",
+      "verified": true,
+      "deployments": {
+        "eth": "0xcA11bde05977b3631167028862bE2a173976CA11",
+        "op": "0xcA11bde05977b3631167028862bE2a173976CA11"
+      }
+    },
+    {
+      "name": "Create2Deployer",
+      "verified": true,
+      "deployments": {
+        "eth": "0xF49600926c7109BD66Ab97a2c036bf696e58Dbc2"
+      }
+    },
+    {
+      "name": "Safe_v130",
+      "verified": true,
+      "deployments": {
+        "eth": "0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552",
+        "op": "0x69f4D1788e39c87893C980c06EdF4b7f686e2938"
+      },
+      "deploymentSalt": "0000000000000000000000000000000000000000000000000000000000000000"
+    },
+    {
+      "name": "SafeL2",
```

Comment:
This doesn't need to be versioned similarly to how its `Safe_v130`?

Reply:
The local contracts binding generation doesn't output bindings for `SafeL2` like it does for `Safe`, so there are no existing bindings being overwritten when this remote generation script is ran. However, that could easily change, and it wouldn't be clear that the bindings script needs to be updated, so I'll version it just to be safe. Thank you for calling this out!

Reply:
I've versioned all the Safe contracts